### DISCUSSION TITLE
Rendering of mathematical expressions (Latex, Mathjax)

### DISCUSSION THREAD

**Discussion by AlFrauch** [flet-dev/flet#5499] [Extensions] [answered]
The ability to render mathematical images (Latex, Mathjax,...) is required 

Markovna does not support mathematical expressions even as https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

For example , the solution on flutter https://pub.dev/packages/flutter_tex

**Comment by DawnMagnet** [comment]
I really need that!

**Comment by taaaf11** [comment]
Got the need for it today

**Comment by Monkez** [comment]
I hope the author adds this feature.

**Comment by Ceravoux** [comment]
+1, this is a useful addition

**Comment by youssefadly237** [comment]
I second that

**Comment by 7576457** [comment]
Looks like people want this :)

**Reply by youssefadly237** [reply]
thanks for #6069 man 🙏

**Comment by ndonkoHenri** [comment, marked as answer]
Latex is now supported in `Markdown` control as from [0.80.5](https://github.com/flet-dev/flet/releases/tag/v0.80.5), thanks to @7576457 !
